lib/backend/dynamo/configure_test.go

```diff
@@ -39,6 +39,7 @@ func TestContinuousBackups(t *testing.T) {
 	b, err := New(context.Background(), map[string]interface{}{
 		"table_name":         uuid.New() + "-test",
 		"continuous_backups": true,
+		"region":             "us-west-2",
```

Comment:
`us-west-2` is still mentioned here and in `TestAutoScaling`.

Reply:
Removed

Reply:
Removed